docs/notes/2025.11.10.10.55.51.md

```
@@ -0,0 +1,421 @@
+    "tags": ["#memory","#design"],
+    "schema": "mem@1"
+  },
+
+  "content": {
+    "text": "raw markdown or plain text (or code)",
+    "sections": [
+      {"h": "Context", "body": "..."},
+      {"h": "Decision", "body": "..."}
+    ],
+    "attachments": [{"mime":"text/x-code","path":"...","lang":"ts"}]
+  },
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)